src/SFA.DAS.ForecastingTool.Web/Standards/ApprenticeshipRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.ForecastingTool.Core.Models;
+using SFA.DAS.ForecastingTool.Infrastructure.Services;
+
+namespace SFA.DAS.ForecastingTool.Web.Standards
+{
+    public class ApprenticeshipRepository : IApprenticeshipRepository
```

Comment:
This class doesn't add any value, you could use IGetApprenticeship, and maybe make the CachedRepository use that as well or drop IGetApprenticeship